examples/meson.build

```diff
@@ -0,0 +1,21 @@
+shared_module(
+  'fib',
+  'fib.c',
+
+  name_prefix: '',
+  gnu_symbol_visibility: 'default',
+  c_args: ['-DJS_SHARED_LIBRARY'],
+  dependencies: host_system == 'windows' ? qjs_dep : [],
+  link_args: host_system == 'darwin' ? ['-undefined', 'dynamic_lookup'] : [],
```

Comment:
This shouldn't be necessary with a `shared_module`, because those arguments are explicitly added by Meson for shared_modules.

Reply:
including the `dependencies` line?

Reply:
fixed.